cmd/openshift-install/migrate/azure/eligible.go

```diff
@@ -0,0 +1,23 @@
+package azure
```

Comment:
the file name is `azure/eligible.go` but it only have `privatedns eligible` command, can we rename this file to reflect what it has.